shopfloor_delivery_shipment_mobile/static/wms/src/scenario/delivery_shipment.js

```diff
@@ -0,0 +1,445 @@
+                return this.state.data.on_dock;
+            } else if (
+                this.filter_state === "lading" &&
+                !_.isEmpty(this.state.data.lading)
+            ) {
+                return this.state.data.lading;
+            }
+            return {};
+        },
+        filter_pickings: function(pickings) {
+            let res = pickings;
+            const nameFilter = this.state.data.filter_name;
+            if (nameFilter) {
+                res = _.filter(pickings, pick => {
+                    return pick.name.indexOf(nameFilter) >= 0;
+                });
+            }
+            return res;
+        },
+        shipment: function() {
+            if (_.isEmpty(this.state.data.shipment_advice)) {
+                return {};
+            }
+            return this.state.data.shipment_advice;
+        },
+        getPackOptions: function(pack) {
+            const action = pack.is_done
+                ? () => {
+                      this.unloadPack(pack);
+                  }
+                : null;
+            return {
+                key_title: "name",
+                on_title_action: action,
+                title_action_icon: "mdi-upload",
+            };
+        },
+        getPackColor: function(pack) {
+            const color = pack.is_done ? "screen_step_done" : "screen_step_todo";
+            return this.utils.colors.color_for(color);
+        },
+        getLineOptions: function(line) {
+            const action =
+                line.qty_done == line.quantity
+                    ? () => {
+                          this.unloadLine(line);
+                      }
+                    : null;
+            return {
+                // main: true,
+                key_title: "product.display_name",
+                on_title_action: action,
+                title_action_icon: "mdi-upload",
+                fields: [
+                    {path: "lot.name", label: "Lot"},
+                    {path: "quantity", label: "Qty"},
+                ],
+            };
+        },
+        getLineColor: function(line) {
+            const color =
+                line.qty_done >= line.quantity
+                    ? "screen_step_done"
+                    : "screen_step_todo";
+            return this.utils.colors.color_for(color);
+        },
+        unloadLine: function(line) {
+            this.wait_call(
+                this.odoo.call("unload_move_line", {
+                    shipment_advice_id: this.shipment().id,
+                    move_line_id: line.id,
+                })
+            );
+        },
+        unloadPack: function(pack) {
+            this.wait_call(
+                this.odoo.call("unload_package_level", {
+                    shipment_advice_id: this.shipment().id,
+                    package_level_id: pack.id,
+                })
+            );
+        },
+        getOperationColor: function(pick) {
```

Comment:
please rename all such methods for options to snake case to make them more readable and respect consistency w/ other scenario

Reply:
:thu